front-end/src/components/map/MapDisplay/MapDisplay.jsx

```diff
@@ -134,7 +134,7 @@ const MapDisplay = ({ setLapsCallback, gotoTime }) => {
   // GEOJSON USES LONG, LAT NOT LAT, LONG
 
   const marker = L.icon({
-    iconUrl: '/topdown_outline.png',
+    iconUrl: (isElectron ? __dirname : '') + '/topdown_outline.png',
```

Comment:
I'm fine to update this, I think that the reason that this is the only image that I had to do this for is because it is pathing to it directly instead of using the @assets pathing. @JAWalmsley is there a reason that this path is using the direct pathing instead of using a reference to the object?

Reply:
Moved the assets to the assets folder from public and then used that so now IsElectron is not being used anymore here